eng/pipelines/templates/Tests_On_Linux.yml

```diff
@@ -25,12 +25,4 @@ steps:
     testRunTitle: "NuGet.Client Tests On Linux"
     searchFolder: "$(Build.Repository.LocalPath)/build/TestResults"
     mergeTestResults: "true"
-  condition: "succeededOrFailed()"
-
-- task: PublishBuildArtifacts@1
-  displayName: "Publish Test Hang Dump"
-  inputs:
-    PathtoPublish: "$(Build.Repository.LocalPath)/build/TestResults"
-    ArtifactName: "$(Agent.JobName)"
-    ArtifactType: "Container"
-  condition: "or(failed(), canceled())"
+  condition: "succeededOrFailed()"
```

Comment:
GitHub is showing me this file (and several others) don't have a final newline.  Our .editorconfig file, even in the release-5.11.x branch specifies that it should: https://github.com/NuGet/NuGet.Client/blob/release-5.11.x/.editorconfig#L14

Please consider adding editconfig extensions to whatever editor you're using, so when making changes to the dev branch that it applies automatically.

Reply:
Yes, and also, can we add the new line here since we are touching this script now?

Reply:
Done in https://github.com/NuGet/NuGet.Client/pull/6698/commits/595e63bce1d55d8bca499f60c1d14869b0fa22a2